client/overview/modal/setup-live-payments/index.tsx

```diff
@@ -13,16 +13,39 @@ import { Icon, currencyDollar } from '@wordpress/icons';
 import BlockEmbedIcon from 'components/icons/block-embed';
 import BlockPostAuthorIcon from 'components/icons/block-post-author';
 import './style.scss';
+import { recordEvent } from 'wcpay/tracks';
 
 interface Props {
+	from: string;
+	source: string;
 	onClose: () => void;
 }
 
-const SetupLivePaymentsModal: React.FC< Props > = ( { onClose }: Props ) => {
+const SetupLivePaymentsModal: React.FC< Props > = ( {
+	from,
+	source,
+	onClose,
+}: Props ) => {
 	const handleSetup = () => {
+		recordEvent( 'wcpay_onboarding_flow_setup_live_payments', {
```

Comment:
As mentioned in comment above, this event should be added to the `client/tracks/event.d.ts` file.